rosidl_typesupport_opensplice_c/cmake/rosidl_typesupport_opensplice_c_generate_interfaces.cmake

```diff
@@ -284,9 +284,8 @@ if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
     LIBRARY DESTINATION lib
     RUNTIME DESTINATION bin
   )
-ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${OpenSplice_LIBRARIES})
-
-endif()
 
+  ament_export_libraries(${rosidl_generate_interfaces_TARGET}${_target_suffix} ${OpenSplice_LIBRARIES})
 
-ament_export_include_directories(include)
+  ament_export_include_directories(include)
```

Comment:
Look further up where and when headers are being installed to the include folder. The condition is different. It's probably easiest to do it there.